### ISSUE TITLE
Error: Identifier not found "RT_GROUP_CURSOR"

### ISSUE THREAD

**Issue by rchastain** [bgrabitmap/bgrabitmap#73]
Hello!
I get a compilation error under Windows 64 with BGRABitmap 10.6.3.
```
bgrabitmaptypes.pas(1338,24) Error: Identifier not found "RT_GROUP_CURSOR"
bgrabitmaptypes.pas(1339,24) Error: Identifier not found "RT_BITMAP"
bgrabitmaptypes.pas(1340,24) Error: Identifier not found "RT_GROUP_ICON"
bgrabitmaptypes.pas(1341,33) Warning: Symbol "RT_RCDATA" is deprecated: "Use Windows.RT_RCDATA instead"
bgrabitmaptypes.pas(1342,34) Warning: Symbol "RT_RCDATA" is deprecated: "Use Windows.RT_RCDATA instead"
bgrabitmaptypes.pas(1343,24) Error: Identifier not found "RT_HTML"
bgrabitmaptypes.pas(1344,25) Error: Identifier not found "RT_HTML"
bgrabitmaptypes.pas(1357,17) Warning: Symbol "RT_RCDATA" is deprecated: "Use Windows.RT_RCDATA instead"
bgrabitmaptypes.pas(1369,12) Error: Identifier not found "RT_GROUP_CURSOR"
bgrabitmaptypes.pas(1369,38) Error: Identifier not found "RT_GROUP_ICON"
```
I compile with the option **-dBGRABITMAP_USE_FPGUI**.
Best regards.
Roland

**Comment by circular17** [comment]
Hello Roland,

I saw something similar in the forum but I did not really get what it was about. Thank you for posting it here. Indeed this problem happens on Windows because the constants have been moved from Windows to System unit but only for Linux. So fpGUI that doesn't use the Windows unit cannot find them.

I've fixed it on dev branch by adding those constants in this case.

Best regards,

Johann
